doc/user_guide/profiling.rst

```diff
@@ -414,10 +443,10 @@ Naming Profiling Regions
 ------------------------
 A profile region derives its name from two components:
 
-`module_name`
-    A string identifying the psy-layer containing this 
+``module_name``
+    A string identifying the PSy-layer containing this
     profile node.
-`region_name`
+``region_name``
     A string identifying the invoke containing 
```

Comment:
"...invoke (PSyKAl DSL) or routine (existing code) containing..." and "...within the invoke/routine (where..."

Reply:
Done.